# STRAND
Structure Prediction with Graph Attention Networks for RNA

Overview:
RNA-GAT (RNA Graph Attention) is an innovative bioinformatics project aimed at advancing RNA structure prediction using state-of-the-art deep learning techniques. By leveraging graph attention neural networks (GATs), RNA-GAT offers a novel approach to accurately predict both secondary and three-dimensional (3D) structures of RNA molecules. This repository serves as a comprehensive toolkit for researchers and practitioners in the field of computational biology and structural bioinformatics.

Features:

1. Comprehensive Base Pairing Modeling: Unlike traditional approaches that focus solely on Watson-Crick base pairings, RNA-GAT incorporates a comprehensive modeling framework that considers all possible and diverse types of base interactions. This includes but is not limited to canonical Watson-Crick pairs (A-U, G-C), as well as non-canonical pairs (e.g., G-U, A-G), base triples, base quadruples, pseudoknots, and other tertiary interactions observed in RNA structures.
2. Graph Attention Mechanisms: The graph attention neural network architecture of RNA-GAT enables the effective capture of complex relationships between nucleotides in RNA sequences. By attending to both local and long-range interactions, the model can learn intricate base pairing patterns and structural motifs beyond simple Watson-Crick pairs, leading to more accurate predictions of RNA secondary and tertiary structures.
3. Improved Prediction Accuracy: By considering a broader range of base pairings and tertiary interactions, RNA-GAT achieves higher prediction accuracy compared to conventional methods. The incorporation of non-canonical base pairs and tertiary contacts allows for a more realistic representation of RNA folding landscapes, facilitating the identification of functionally relevant structural features and motifs.
